extensions/quartz/runtime/pom.xml

```diff
@@ -33,7 +33,7 @@
             <exclusions>
                 <exclusion>
                     <groupId>com.zaxxer</groupId>
-                    <artifactId>HikariCP-java6</artifactId>
+                    <artifactId>HikariCP-java7</artifactId>
```

Comment:
I think I would keep excluding the old one and also exclude `HikariCP`.

Reply:
done